zerver/tests/test_auth_backends.py

```diff
@@ -416,10 +440,11 @@ def test_github_backend_do_auth_for_default_auth_failed(self):
                         side_effect=AuthFailed('Not found')), \
                 mock.patch('logging.info'), \
                 mock.patch('zproject.backends.SocialAuthMixin.process_do_auth') as result:
+            self.backend.strategy.session_set('subdomain', 'zulip')
```

Comment:
The commit that adds this is titled "Set the strategy ..." -- should that be "subdomain"?

Reply:
yep, changed.